Automattic/wp-calypso
# New public blog setting

Do not merge until other pieces are in place. 

## Proposed Changes

Adds a new setting to the site's privacy page for wpcom and WoA sites, which opts out the site content from third parties and research partners. 

More info: peGwbA-1tL-p2

## Testing Instructions



On a wpcom simple and WoA site: 
- Head to /settings/general/
- Toggle `Prevent third party sharing` and save. Reloading the page should persist. 
- When the "Prevent third party sharing" option is checked, so should this one. It should not be possible to deselect the new setting when "Discourage search engines" is selected. 

You should not see this setting on a self-hosted Jetpack site. 

## Pre-merge Checklist
